src/hotspot/share/gc/z/zPageAllocator.hpp

```diff
@@ -169,6 +169,8 @@ class ZPageAllocator {
   void handle_alloc_stalling_for_old(bool cleared_soft_refs);
 
   void threads_do(ThreadClosure* tc) const;
+
+  void cleanup_unused();
```

Comment:
This should not be addressed in this PR, but I find "cleanup_unused" name pretty undescriptive, both here and in `CollectedHeap` in general. It wasn't obvious to me that this is something used only by CRaC. I would propose `cleanup_before_checkpoint`, for example.